MODULE.bazel.lock

```diff
@@ -2248,7 +2248,7 @@
       "general": {
         "bzlTransitiveDigest": "/h6B2Wl9PVY+Bv0PAxzkZUmm947eWCyYthDYT5CPX+k=",
         "accumulatedFileDigests": {
-          "@@//src/test/tools/bzlmod:MODULE.bazel.lock": "e7caa4d6182abd31642ac06dfc30a4501a998924fbd68b59eced1e1cf836da5f",
+          "@@//src/test/tools/bzlmod:MODULE.bazel.lock": "12e9ba281d672e202359d8f7f740b30fca190012aaa24a7ac04c547233534e74",
```

Comment:
I don't know what's current best practice, but I'm guessing MODULE.bazel.lock should not get merged with arbitrary PR. Please remove it.

Reply:
@comius This is probably necessary: Changing `unix_cc_configure.bzl` changes the transitive bzl digest of the `local_config_cc` module extension used in `bazel_tools`, which in turn causes the `MODULE.bazel.lock` for the tools module (used in tests) to change, which in turn causes the root module lockfile as the `bazel_build_deps` extension parses this lockfile.

Reply:
I thought I wasn't up to date. Thanks for clarification.